src/config.rs

```diff
@@ -0,0 +1,78 @@
+use human_size::Bytes;
+use std::str::FromStr;
+use slog::{self, Logger, Filter, DrainExt, Level};
+use slog_term;
+
+fn max_assets_per_document(logger: &slog::Logger) -> u8 {
+    let default = 20;
+    match ::std::env::var("PAPERS_MAX_ASSETS_PER_DOCUMENT").map(|max| max.parse()) {
+        Ok(Ok(max)) => max,
+        Ok(Err(_)) => {
+            warn!(logger,
+                  "Unable to parse PAPERS_MAX_ASSETS_PER_DOCUMENT environmental variable");
+            default
+        }
+        _ => default,
+    }
+}
+
+pub fn is_debug_active() -> bool {
+    match ::std::env::var("PAPERS_LOG_LEVEL") {
+        Ok(ref level) if level.contains("debug") => true,
+        _ => false,
+    }
+}
+
+#[derive(Debug)]
+pub struct Config {
+    /// TODO: document this here
```

Comment:
`A long secret that is used in the Authorization header to authenticate a request against papers`